app/javascript/tapp/fetch.js

```diff
@@ -70,33 +72,28 @@ function putHelper(URL, body) {
 
 const getApplicants = () =>
     getHelper('/applicants')
-        .then(resp => resp.json())
-        .then(onFetchApplicantsSuccess)
-        .catch(defaultFailure);
+        .then(resp => resp.json().catch(defaultFailure))
```

Comment:
What does `json()` return if it fails? Do we need to pass a message to `defaultFailure` manually?